### PR TITLE
Feature/mobile hosting cx 1174

### PR DESCRIPTION
**Pull request by hjheath**

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-223-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-223-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-223-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-223-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-223-pr-onfido-sdk-ui-onfido.surge.sh

File: release_guidelines.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ An example `<version>` could be `2.0.0-rc.1`
 
 * Create a release branch: `release/<version>`. Use the final version rather than a release candidate in the branch name.
 * Update the version in `package.json`
+* Add the version to the hash in src/components/utils/versionMap.js
```
Comment (dawiddylowicz) [inline release_guidelines.md]: Could you make the path enhanced with `? Like this: `src/components/utils/versionMap.js`

Comment (dawiddylowicz) [reply release_guidelines.md]: Unless it's no longer valid, as I can see the file has been removed

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-223-pr-onfido-sdk-ui-onfido.surge.sh

File: release_guidelines.md [hunk 2]
```diff
@@ -13,7 +13,13 @@ An example `<version>` could be `2.0.0-rc.1`
 * Create a release branch: `release/<version>`. Use the final version rather than a release candidate in the branch name.
 * Update the version in `package.json`
 * Update the change log following [this](http://keepachangelog.com/)
+* Increment BASE_36_VERSION in `webpack.config.babel.js` e.g. 00 => 01
+  * You only need to do this once per release, *not* per release candidate
 * Build the dist files with `npm run build`
+* Run `aws s3 sync . s3://onfido-assets-production/web-sdk/<base36 version>/ --exclude "*.html" --exclude "*.map" --acl public-read --delete
+
```
Comment (dawiddylowicz) [inline release_guidelines.md]: Empty lines here

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-223-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**